deploy/1_deploy_usmfum.js

```diff
@@ -1,41 +1,33 @@
-
 const chainlinkAddresses = {
-  '1' : '0x5f4eC3Df9cbd43714FE2740f5E3616155c5b8419',
-  '42' : '0x9326BFA02ADD2366b30bacB125260Af641031331',
+  1: '0x5f4eC3Df9cbd43714FE2740f5E3616155c5b8419',
+  42: '0x9326BFA02ADD2366b30bacB125260Af641031331',
 }
 const compoundAddresses = {
-  '1' : '0x922018674c12a7f0d394ebeef9b58f186cde13c1',
-  '42' : '0x0000000000000000000000000000000000000000',
-}
-const ethUsdtAddresses = {
-  '1' : '0x0d4a11d5eeaac28ec3f61d100daf4d40471f1852',
-  '42' : '0x0000000000000000000000000000000000000000',
+  1: '0x922018674c12a7f0d394ebeef9b58f186cde13c1',
+  42: '0x0000000000000000000000000000000000000000',
 }
 const usdcEthAddresses = {
-  '1' : '0xb4e16d0168e52d35cacd2c6185b44281ec28c9dc',
-  '42' : '0x0000000000000000000000000000000000000000',
-}
-const daiEthAddresses = {
-  '1' : '0xa478c2975ab1ea89e8196811f51a7b7ade33eb11',
-  '42' : '0x0000000000000000000000000000000000000000',
```

Comment:
I think it's worth keeping the USDT & DAI addresses around - we could need them sometime (possibly even in a hurry!).

Reply:
If that's the case we could then move them to their own `adddresses/contracts/constants` files, to both keep the linter happy and so they're reusable